build.js

```diff
@@ -786,7 +790,9 @@ const copyResources = async () => {
     ['@mdi/font/fonts/materialdesignicons-webfont.woff2', 'fonts']
   ];
   for (const [srcInModules, destInGenerated, newFilename] of resources) {
-    const src = require.resolve(srcInModules);
+    const src = fileURLToPath(
+      new URL(`./node_modules/${srcInModules}`, import.meta.url)
+    );
```

Comment:
```suggestion
    const src = `./node_modules/${srcInModules}`;
```

Reply:
I've found that it's actually better if we continue to use URL objects.  `fs` supports URL objects natively (and in turn, so does `fs-extra`), so we won't need to use `fileURLToPath` everywhere.  However, if we pass a string as a relative file path, `.` is resolved to the current working directory, which we cannot always trust to be the root of the project.

Reply:
Oh, cool, that does make things nicer. Docs at https://nodejs.org/api/fs.html#fs_file_url_paths.